src/audio/copier/copier.c

```diff
@@ -636,6 +689,40 @@ static int do_module_copy(struct copier_data *cd, struct comp_dev *dev)
  * from single stream (for output gateways) so such gateways work kind of like a single
  * gateway, i.e., produce/consume single stream.
  */
+
+static int copier_process(struct processing_module *mod,
```

Comment:
this will likely run into compilation issues because this function is unused. Either you need to call this from copier_copy() or you need convert to module_interface in this patch